tests/br_rawkv/client.go

```diff
@@ -32,8 +33,8 @@ var (
 		"kv pairs are separated by commas, key and value in a pair are separated by a colon")
 )
 
-func createClient(addr string) (*tikv.RawKVClient, error) {
-	cli, err := tikv.NewRawKVClient([]string{addr}, config.Security{
+func createClient(addr string) (*rawkv.Client, error) {
+	cli, err := rawkv.NewClient(context.TODO(), []string{addr}, config.Security{
```

Comment:
using `context.TODO()` means this will change to accept a generic context in future versions?

Reply:
Yes, if in the future we want to apply context to our tests, a context can be passed from the upper level to replace `context.TODO()`.